crates/storage/src/collections/vec/fuzz_tests.rs

```diff
@@ -83,3 +83,105 @@ crate::fuzz_storage!("vec_1", StorageVec<u32>);
 crate::fuzz_storage!("vec_2", StorageVec<Option<Pack<Option<u32>>>>);
 crate::fuzz_storage!("vec_3", StorageVec<(bool, (u32, u128))>);
 crate::fuzz_storage!("vec_4", StorageVec<(i128, u32, bool, Option<(u32, i128)>)>);
+
+#[quickcheck]
+fn fuzz_binary_search(mut std_vec: Vec<i32>) {
+    // given
+    if std_vec.len() == 0 {
+        return
+    }
+    let original_std_vec = std_vec.clone();
+    std_vec.sort();
+    let ink_vec = StorageVec::from_iter(std_vec.clone());
+
+    for x in original_std_vec {
+        // when
+        let index = std_vec
+            .binary_search(&x)
+            .expect("`x` must be found in `Vec`") as u32;
+        let index_sv = ink_vec
```

Comment:
```suggestion
        let ink_index = ink_vec
```

Maybe change this for consistency with the other tests?